visallo/visallo
# fix users not seeing real time additions of new work product in share…

Testing Instructions:
```
Test 1:
Load a product then view the network tab. There should only be one request where `product?productId=.

As User A create a case and add a graph product with a vertex.
Share the case to User B
As User A load a different case.
As User B, in a separate session, load the shared case and move the vertex.
As User A check the network panel shows a product?productId=<productId> request with an extendedData: false header
```
```
Test 2:
Have users test01 and test02
Share test01's case with test02 with edit access
As test01 create a graph and add data
As test02 on the same case, add another graph and add data
View available work products as test01, you should see the new graph work product
```

Points of Regression: Switching cases/selecting work products. Product list not updating correctly

NO CHANGELOG
